src/kubectlUtils.ts

```diff
@@ -0,0 +1,107 @@
+import * as vscode from "vscode";

```

Comment:
Should this module have unit tests?

Reply:
The methods on this module are just invoking kubectl command, don't have much special logic. I would unit tests on lower priority.